src/personality_test1.c

```diff
@@ -382,10 +383,10 @@ static void PrintPersonalityTypeDescription(void)
 static void PersonalityTest_DisplayStarterSprite(void)
 {
     s32 starterID;
-    OpenedFile *faceFile;
+    struct FileMonPortraits *faceFile;
```

Comment:
I don't like the idea of having N different OpenedFile variants for various data structures, instead of casting the data pointer into the proper structure

Reply:
I'm fine with either. I'll change it to casting as it seems that's how it's been written so far.

Reply:
Changed to a cast